concrete/controllers/single_page/download_file.php

```diff
@@ -54,7 +54,8 @@ public function view($fID = 0, $rcID = null)
             $file = File::getByID($fID);
 
             if ($file instanceof FileEntity && $file->getFileID() > 0) {
-                $rcID = $this->app->make('helper/security')->sanitizeInt($rcID);
+                $rcID = intval($rcID);
```

Comment:
Just a side note: `(int) $rcID` is a tiny bit faster than `intval($rcID)`

Reply:
aefef65532a74629673e19613f4dc8003d77ed31